### DISCUSSION TITLE
using v3 with react

### DISCUSSION THREAD

**Discussion by aphleps** [djipco/webmidi#203] [Migration] [answered]
I've been using v2.5 with react but didn't manage to get v3 working. After scaffolding a new project with create-react-app I installed webmidi 3.x using npm. When I try to call the API I get the following error.

```
./node_modules/webmidi/dist/cjs/webmidi.cjs.min.js
Module not found: Can't resolve 'perf_hooks' in '/Users/alex/code/react-midi-simple/node_modules/webmidi/dist/cjs'
```

**Comment by djipco** [comment, marked as answer]
Thanks for reporting the problem. I was able to reproduce it. I just published v3.0.3 which should fix the issue. Can you please confirm that it does resolve the problem for you?

**Reply by aphleps** [reply]
wow that was fast. it works! thank you very much

**Reply by djipco** [reply]
My pleasure! 

By the way, I added a `react` subfolder in the [`examples`](https://github.com/djipco/webmidi/tree/master/examples) directory. I created an example project with `create-react-app` but it is kind of useless. If you ever want to contribute a proper WEBMIDI.js starter example for React users, that would be awesome. 

Cheers!
